kroma-chain-ops/genesis/genesis.go

```diff
@@ -62,8 +62,7 @@ func NewL2Genesis(config *DeployConfig, block *types.Block) (*core.Genesis, erro
 		ShanghaiTime:                  config.CanyonTime(block.Time()),
 		CancunTime:                    config.EcotoneTime(block.Time()),
 		EcotoneTime:                   config.EcotoneTime(block.Time()),
-		// TODO(seolaoh): uncomment this when geth updated
-		// KromaMPTTime:                  config.KromaMPTTime(block.Time()),
+		KromaMPTTime:                  config.KromaMPTTime(block.Time()),
```

Comment:
How about changing this to `KromaMPTBlock`? It will induce changes in geth, but I think it will be more coherent with the handling of Bedrock fork.

Reply:
We've been doing upgrades by timestamp, not block number, so i thought it was better to make sure there was no confusion.